developer_guide/installation/system_requirements/setup_requirements_debian8_php7.rst

```diff
@@ -0,0 +1,109 @@
+Setup System Requirements on Debian 8 Jessie with PHP7
+======================================================
+
+.. warning::
+
+    We continued our effort regarding Akeneo PIM PHP7 support. PHP7 is now usable in experimental mode for both CLI and Web.
+    Experimental means that we manage to install and use the PIM but due to missing tests in our functional matrix we can't commit to support it.
+
+Here is a quick guide to setup the :doc:`system_requirements` on Debian 8 Jessie with PHP7. It assumes you have already followed the guide :doc:`setup_requirements_debian8` and have the PIM working with PHP5.
+
+.. note::
+
+    Please perform the following commands as root.
+
+System installation
+-------------------
+
+.. _dotdeb: https://www.dotdeb.org/instructions/
+
+To install PHP7 on Debian 8 Jessie, you can use the the dotdeb_ repositories. Edit your ``/etc/apt/sources.list`` and add the following:
+
+.. code-block:: text
+
+    deb http://packages.dotdeb.org jessie all
+    deb-src http://packages.dotdeb.org jessie all
+
+Then add the dotdeb key to your know keys:
+
+.. code-block:: bash
+    :linenos:
+
+    wget https://www.dotdeb.org/dotdeb.gpg
+    apt-key add dotdeb.gpg
+
+Now install the required dependencies:
+
+.. code-block:: bash
+    :linenos:
+
+    $ apt-get update
+    $ apt-get install mysql-server apache2 libapache2-mod-php7.0 php7.0-cli php7.0-apcu php7.0-mcrypt php7.0-intl php7.0-mysql php7.0-curl php7.0-gd php7.0-xml php7.0-zip
```

Comment:
sudo or # here too
Also add php7.0-apcu-bc Without it, the PIM doesn't work in prod mode.